config/admin_set_map.yml

```diff
@@ -0,0 +1,14 @@
+#This file is in the order of `ModelName: "Admin Set Name"`
+#If changes are being made to this file, please follow this order
+
+Article: "Article"
+Graduate Thesis Or Dissertation: "Graduate Thesis Or Dissertation"
```

Comment:
The keys are all humanized.. you want to remove the spaces to map them as models.